packages/@aws-cdk/integ-runner/package.json

```diff
@@ -54,7 +54,6 @@
   "devDependencies": {
     "@aws-cdk/cdk-build-tools": "0.0.0",
     "@aws-cdk/core": "0.0.0",
-    "@aws-cdk/integ-tests": "0.0.0",
```

Comment:
This is to avoid a circular dependency with `integ-tests`. It's okay because the only usage is in the language tests which are currently run manually.